## 1.1 Extract Information From Your Own Transaction

In [29]:
import json
import requests

In [28]:
answer = requests.get("https://api.whatsonchain.com/v1/bsv/main/block/height/737045")
genesis_block = json.loads(answer.text)
print(genesis_block['txcount'])
print(genesis_block['time'])
print(genesis_block['totalFees'])
print(genesis_block['confirmations'])
print(genesis_block['miner'])

1645
1651098456
0.01747634999999992
1
SVpool


In [18]:
import datetime
timestamp = datetime.datetime.fromtimestamp(genesis_block['time'])
print(timestamp.strftime('%Y-%m-%d %H:%M:%S'))

2022-04-27 23:27:36


### 
miner is the node of this transaction，the first transaction of this block.

timestamp is when this transaction is confirmed at 2022-04-27 23:27:36

confirmations is how many legers have confirmed this transaction.because of this few amount of piece, only one leger confirmed this transaction at the moment. After 1 hour I checked again,this number aggregated to 11, which means this transaction is irreversible.

Total Fee is the transaction Fee on this block combine.

txcount is the total number of transactions happened in this block up until now, which is 1645.

## 1.2 Create a Bitcoin Wallet

In a seperate file